public/locales/en/translations.json

```diff
@@ -424,6 +424,20 @@
           "value": "Value",
           "valuePlaceholder": "Enter value"
         },
+        "actions":{
+          "action": "Action",
+          "emailActionLabel": "Send email",
+          "emailAddresses": "Email addresses",
+          "enterEmail": "Press enter to add email recipient",
+          "emailComments": "Email Comments",
+          "enterEmailComments": "Enter comments for the email recipient(s)",
+          "emailSubject": "Email Subject",
+          "enterEmailSubject": "Azure IoT Remote Monitoring alert: <Rule Name> was triggered",
```

Comment:
Do we need to render this as is -> <Rule Name>?

Reply:
Yes this is the prompt text, to show the rule name will be filled in there